src/EditorFeatures/CSharpTest/NavigateTo/NavigateToTests.cs

```diff
@@ -25,6 +27,26 @@ public class NavigateToTests : AbstractNavigateToTests
         protected override TestWorkspace CreateWorkspace(string content, ExportProvider exportProvider)
             => TestWorkspace.CreateCSharp(content, exportProvider: exportProvider);
 
+        private readonly static PatternMatch s_emptyExactPatternMatch = new PatternMatch(PatternMatchKind.Exact, true, true, ImmutableArray<Span>.Empty);
```

Comment:
isn't this exactly same as the test above? can this be moved to some shared code?

Reply:
is this and the test above (interactive navigate to test) actually test different thing? except I guess some workspace options? can 2 be merged?

Reply:
Yes, these 2 files have many duplicate tests.  It will require a little bit to refactor them into one so I filed a separate issue for this https://github.com/dotnet/roslyn/issues/24318